comp/core/workloadmeta/types.go

```diff
@@ -782,19 +897,38 @@ func (d KubernetesDeployment) String(verbose bool) string {
 
 var _ Entity = &KubernetesDeployment{}
 
+// ECSTaskKnownStatusStopped is the known status of an ECS task that has stopped.
+const ECSTaskKnownStatusStopped = "STOPPED"
+
+// ECSTaskTags is a map of tags for an ECS task.
+type ECSTaskTags map[string]string
+
+// ContainerInstanceTags is a map of tags for an ECS container instance.
+type ContainerInstanceTags map[string]string
```

Comment:
should have have only one type: `MapTags`? I dont see the added value to have 2 types here